collectors/feed/collectors.py

```diff
@@ -0,0 +1,281 @@
+import logging
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+from itertools import chain
+
+import feedparser
+import requests
+import yaml
+from requests.exceptions import RequestException
+
+from osidb.models import Snippet
+
+from ..framework.models import KnownFeedEntry
+from .constants import (
+    CHROMIUM_REGEX,
+    FEEDS_FILE,
+    MAX_COLLECTOR_WORKERS,
+    REQUEST_TIMEOUT,
+)
+from .keywords import check_keywords
+from .models import FeedEntry, SkippedFeedEntry
+from .utils import find_cves, find_cvss, find_cwes, html_to_text
+
+log = logging.getLogger(__name__)
+
```

Comment:
Open problem (not related to this line, just putting it in a place that has discussion threading):

Bandit (as run via the githook) fails without an explanation.

Reply:
Didn't end up adding such a comment, because I think reading the code is much more helpful, than any comment would be. It could also devolve into a screen-long risk analysis. If you have a better idea, please let me know.